CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.2)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
I am not sure why, but I wasn't able to fix dynamic versioning without this requirement.